src/librustc_resolve/lib.rs

```diff
@@ -953,6 +981,19 @@ impl<'a> ModuleS<'a> {
         assert!(self.pub_glob_count.get() > 0);
         self.pub_glob_count.set(self.pub_glob_count.get() - 1);
     }
+
+    fn is_successor_of(&self, ancestor: Module<'a>) -> bool {
```

Comment:
To keep things as simple as possible, I would report all items in the current crate and all accessible items in external crates. If we don't need to distinguish between accessible and inaccessible items in the current crate, we wouldn't need this method.